lib/Math/Spiral.pm

```diff
@@ -126,7 +152,7 @@ sub new {
   my $this={};
   foreach(qw(a b t_inc t_cb)){ $this->{$_}=$args{$_} }
   foreach(qw(x xmin y ydir ymin t)){ $this->{$_}=0; }
-  foreach(qw(xdir xmax ymax t_inc)){ $this->{$_}||=1; }
```

Comment:
This is the only functional change to the code. It would be better if all the arguments had a default (except the callback).  This fix makes that happen.